assets.md

```diff
@@ -370,12 +370,8 @@ This GitHub App addresses two common issues:
    git config user.email 197425009+otelbot@users.noreply.github.com
    ```
 
-   It is recommended to push to branch names that start with `otelbot/`, and to add a branch protection
-   rule for `otelbot/**/*` with the same setup as documented for
-   [`dependabot/**/*`](docs/how-to-configure-new-repository.md#branch-protection-rule-dependabot).
-
-   > [!WARNING]
-   > Branch protection rule **ordering** matters, so you will need to delete the `**/**` branch protection rule temporarily, then add the `otelbot/**/*` branch protection rule, then add back the `**/**` branch protection rule.
```

Comment:
this is one of the reasons that the `**/**` branch protection rule complicates life

rulesets resolve this complication, but we can't move to rulesets because of https://github.com/orgs/community/discussions/61107#discussioncomment-6569780